web/packages/plugins/ui/sima-application-input/src/EditSimaApplicationInput.tsx

```diff
@@ -23,17 +26,47 @@ import {
 import { NotificationManager } from 'react-notifications'
 import {
   Column,
+  Container,
   EditInputEntity,
   GroupWrapper,
   HeaderWrapper,
   Row,
 } from './components'
 import _ from 'lodash'
 
+const ReadOnlyPathTextField = (props: { path: string }) => {
+  return (
+    <Tooltip
+      title={truncatePathString(props.path) === props.path ? '' : props.path}
+    >
+      <Input
```

Comment:
Input fields have readOnly property. Maybe also use that to disable the possibility to write/change anything in the fields.